common/djangoapps/dark_lang/middleware.py

```diff
@@ -67,88 +29,23 @@ class DarkLangMiddleware(object):
     using the django admin site.
     """
 
-    @property
-    def released_langs(self):
-        """
-        Current list of released languages
-        """
-        language_options = DarkLangConfig.current().released_languages_list
-        if settings.LANGUAGE_CODE not in language_options:
-            language_options.append(settings.LANGUAGE_CODE)
-        return language_options
-
     def process_request(self, request):
         """
         Prevent user from requesting un-released languages except by using the preview-lang query string.
         """
         if not DarkLangConfig.current().enabled:
             return
 
-        self._clean_accept_headers(request)
+        Darklang().clean_accept_headers(request)
```

Comment:
Moving this method back into this class will allow you to just call self.clean_accept_headers.